site/src/component/SideInfo/SideInfo.tsx

```diff
@@ -146,7 +146,9 @@ const SideInfo: FC<SideInfoProps> = (props) => {
     <div className="side-content-wrapper">
       <div className="side-info">
         <div>
-          <h2>{props.name}</h2>
+          <div className="side-info-name">
```

Comment:
so there used to be a wrapper div here? or was the (flex) style previously applied to the div without a class name?